### ISSUE TITLE
Skclusive Desktop Application for Windows

### ISSUE THREAD

**Issue by naeemahmadV** [skclusive/Skclusive.Material.Docs#20]
![error-on-desktop-app](https://user-images.githubusercontent.com/9449604/118540291-1619b080-b76e-11eb-9d3c-3528cbdf590b.JPG)

When i try to run desktop application it was showing some Error related to Microsoft.Bcl.AsyncInterfaces but i have resolve it by install nuget package.

after face the issue can you please check the and let me know how we can resolve it.

**Comment by naeemahmadV** [comment]
Hi Team did you get chance to check this query !!

**Comment by skclusive** [comment]
yes. installing Microsoft.Bcl.AsyncInterfaces is correct approach.

**Comment by naeemahmadV** [comment]
but face this issue can you confirm what is missing here !

**Comment by skclusive** [comment]
i do not have windows to reproduce. it would be due to target framework version related.

**Comment by naeemahmadV** [comment]
I'm still facing the same issue can you guide me how we can resolve this. there is one urgent task i need to check this framework can run on Windows OS
